pom.xml

```diff
@@ -1715,6 +1715,7 @@
             <!-- Docker and Kubernetes (not part of the distribution) -->
             <exclude>kubernetes/**</exclude>
             <exclude>docker/**</exclude>
+            <exclude>helm/**</exclude>
```

Comment:
why do we need this?

Reply:
I created a hyper link of helm from root

Reply:
Ref: https://github.com/apache/pinot/issues/11647

Reply:
ah. OK